clusterloader2/pkg/prometheus/manifests/exporters/kube-state-metrics/0namespace.yaml

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
here's a blank line that probably can be removed.

Reply:
Nice catch, thanks!